### ISSUE TITLE
skip test_udp.ReactorShutdownInteraction for non-UDP-supporting reactors

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2150]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| PenguinOfDoom reported|
|-|-|
|Trac ID|trac#2150|
|Type|enhancement|
|Created|2006-10-03 23:10:01Z|

```
Index: twisted/test/test_udp.py
===================================================================
--- twisted/test/test_udp.py    (revision 18379)
+++ twisted/test/test_udp.py    (working copy)
@@ -704,6 +704,7 @@
 
 if not interfaces.IReactorUDP(reactor, None):
     UDPTestCase.skip = "This reactor does not support UDP"
+    ReactorShutdownInteraction.skip = "This reactor does not support UDP"
 if not hasattr(reactor, "connectUDP"):
     OldConnectedUDPTestCase.skip = "This reactor does not support connectUDP"
 if not interfaces.IReactorMulticast(reactor, None):

```

<details><summary>Searchable metadata</summary>

```
trac-id__2150 2150
type__enhancement enhancement
reporter__PenguinOfDoom PenguinOfDoom
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__ 
time__1159917001000000 1159917001000000
changetime__1159920111000000 1159920111000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)<a name="note_1"></a>|@jerub set owner to PenguinOfDoom|
|-|-|

Okay, please merge.

**Comment by twisted-trac** [comment]
|<img alt="pahan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|pahan set status to `closed`|
|-|-|

(In [18380]) Skip ReactorShutdownInteraction if there's no UDP support

Author: pahan
Reviewer: jerub
Fixes [#2150](https://github.com/twisted/twisted/issues/2150)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
